modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -33,7 +33,29 @@ This {release} version of the software and programming language should not be us
 
 == Highlights of what's new in {release}
 
-The {release} release primarily includes user-visible new features, internal enhancements, and fixes and updates to the version of {proglang} and the {proglang} base library that are supported. 
+The {release} release primarily includes internal enhancements, and fixes and updates to {proglang}, the {proglang} base library, and Candid. 
+
+The most significant new features and fixes include the following:
+
+* A fix to `+dfx upgrade+` that prevents a `+main+` thread panicked when `+Option::unwrap()+` is called on a `+None+` value. 
+* A fix to the name of the language service command to restore its compatibility with IDEs. 
+* Candid has been updated to support more Rust built-in types and with support for reference types.
```

Comment:
```suggestion
```
This is not related to dfx

Reply:
Do you really want it removed or just labeled as a Candid change? I think people are going to want some type of release notes or changelog for Candid updates eveentually.

Reply:
Candid has its own release notes here: https://github.com/dfinity/candid/blob/master/Changelog.md, and it's a public repo, so people can already see that. It makes sense to include it here if the change affects dfx. In this case, it's only a Rust crate change, it may cause more confusion to include in the dfx release notes.

Eventually, I would expect to have a separate release notes for Motoko when the repo becomes public, so that we can decouple more complexities around release notes.